packages/icons/vaadin-icons.js

```diff
@@ -3,5 +3,5 @@
  * Copyright (c) 2015 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@polymer/iron-icon/iron-icon.js';
```

Comment:
I'm a bit puzzled about the existence of this file, but I guess it's there for legacy reasons?

Reply:
Yes, some demos might import this file, so let's keep it for now.